docs/start/configure-qiskit-local.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Configure Qiskit locally
+description: Configure Qiskit on your local machine
+
+---
+# Configure Qiskit locally
+
+After Qiskit is installed and running, there are some optional steps you can take to change the default Qiskit behavior.
+
+## User configuration file
+
+The main location for local configuration of Qiskit is the user configuration file. This is an .ini-format file that can be used to change Qiskit default settings.
+
+Example:
+
+ ```text
+[default]
+circuit_drawer = mpl
+circuit_mpl_style = default
+circuit_mpl_style_path = ~:~/.qiskit
+state_drawer = hinton
+transpile_optimization_level = 3
+parallel = False
+num_processes = 15
+ ```
+
+By default this file is in `~/.qiskit/settings.conf` but the path can be overridden with the QISKIT_SETTINGS environment variable.
+
+## Available options
+
+- `circuit_drawer`: Changes the default backend for the circuit drawer. It can be set to `latex`, `mpl`, `text`, or `latex_source`. When the output kwarg is not explicitly set, this drawer backend is used.
+- `circuit_mpl_style`: The default style sheet used for the mpl output backend for the circuit drawer. Valid values are `default` or `bw`.
+- `circuit_mpl_style_path`: The paths to have the circuit drawer use to look for JSON style sheets when using the mpl output mode.
+- `state_drawer`: This is used to change the default backend for the state visualization draw methods. Valid values are `repr`, `text`, `latex`, `latex_source`, `qsphere`, `hinton`, or `bloch`. When the output kwarg is not explicitly set on the :meth:`~qiskit.quantum_info.DensityMatrix.draw` method, the specified output method is used.
+- `transpile_optimization_level`: Change the default optimization level for :func:`~qiskit.compiler.transpile` and :func:`~qiskit.execute.execute`. Specify an integer 0-3.
+- `parallel`: Whether Python multiprocessing is enabled for operations that support running in parallel.  For example, transpilation of multiple :class:`~qiskit.circuit.QuantumCircuit` objects. This setting can be overridden by the QISKIT_PARALLEL environment variable. Specify a boolean value.
+- `num_processes`: The maximum number of parallel processes to launch for parallel operations if parallel execution is enabled. This setting can be overridden by the QISKIT_NUM_PROCS environment variable. Specify an integer greater than 0. 
+
+<Admonition type="note">
+    * Circuit drawer settings apply to :meth:`qiskit.circuit.QuantumCircuit.draw` and :func:`qiskit.visualization.circuit_drawer`.
+    * State visualization draw methods are :meth:`qiskit.quantum_info.Statevector.draw` and :meth:`qiskit.quantum_info.DensityMatrix.draw`
+</Admonition>
+
+## Environment variables
```

Comment:
I wonder if environment variables should be in code font? Do you know if there's a standard @beckykd @javabster ?

Reply:
I think its fine as is :)

Reply:
I looked at IBM Style and there's no guidance, but then I looked at some other IBM files that describe environment variables and none of them have special formatting.